tests/README.md

```diff
@@ -8,6 +8,12 @@ This project uses unit, smoke and integration tests with Python files and notebo
 
 For more information, see a [quick introduction to unit, smoke and integration tests](https://miguelgfierro.com/blog/2018/a-beginners-guide-to-python-testing/). To manually execute the unit tests in the different environments, first **make sure you are in the correct environment as described in the [SETUP.md](../SETUP.md)**. 
 
+AzureML is also used to run the existing unit, smoke and integration tests as-is. AzureML benefits include managing the compute environment by automatically turning it on/off, scaling, automatic logging of artifacts from test runs and more. Azure DevOps is used as a control plane to provide information to scripts to configure and run the tests as-is on AzureML.  A separate set of pipelines was created to run the tests on AzureML and parameters to configure AzureML are defined in the pipeline yml files. There are two scripts used with each pipeline:
```

Comment:
how can we make the AzureML tests more explicit? Do we have any metrics of test time on a normal VM vs AzureML? In the past we were thinking of doing a blog post about this. Maybe we can have a section explaining the AzureML tests and adding the comments there.